### PR TITLE
Vbarrera devel

### PR DESCRIPTION
**Pull request by vbarrera**
A few tweaks to a pair of plots around the interestingGroup options.

### CODE REVIEW

File: R/methods-plotCorrelationHeatmap.R [hunk 1]
```diff
@@ -124,7 +124,8 @@ setMethod(
         method = "pearson",
         genes = NULL,
         samples = NULL,
-        title = NULL) {
+        title = NULL,
+        interestingGroups= NULL) {
```
Comment (lpantano) [inline R/methods-plotCorrelationHeatmap.R]: Add interestingGroups to the doc at the beginning of this file. line: 24.

**Comment by mjsteinbaugh** [comment]
Don't merge this yet – needs to get rebased with master

**Review by mjsteinbaugh — CHANGES_REQUESTED** [review]
This branch is behind master

**Comment by vbarrera** [comment]
I'll make the changes over the latest version.

Closing the pull request.

**Pull request closed without merging**